src/guard.md

```diff
@@ -21,6 +24,7 @@ if
 
```

Comment:
isAuthorized returns false if `Authority == 0`, so this can't pass
https://github.com/dapphub/ds-auth/blob/master/src/auth.sol#L63